src/components/ConversationSettings/ListableSettings.vue

```diff
@@ -97,30 +100,50 @@ export default {
 		}
 	},
 
+	beforeDestroy() {
+		if (this.lastNotification) {
+			this.lastNotification.hideToast()
+			this.lastNotification = null
+		}
+	},
+
 	methods: {
+		async toggleListableUsers(event) {
+			await this.saveListable(event.target.checked ? this.LISTABLE.USERS : this.LISTABLE.NONE)
+		},
+
+		async toggleListableGuests(input) {
+			await this.saveListable(event.target.checked ? this.LISTABLE.ALL : this.LISTABLE.USERS)
+		},
+
 		async saveListable(listable) {
-			this.$emit('input', listable.value)
+			this.$emit('input', listable)
 			if (!this.token) {
-				this.listable = listable.value
+				this.listable = listable
 				return
 			}
 			this.isListableLoading = true
 			try {
 				await this.$store.dispatch('setListable', {
 					token: this.token,
-					listable: listable.value,
+					listable: listable,
 				})
 
-				if (listable.value === CONVERSATION.LISTABLE.NONE) {
-					showSuccess(t('spreed', 'You made the conversation invisible'))
-				} else if (listable.value === CONVERSATION.LISTABLE.USERS) {
-					showSuccess(t('spreed', 'You made the conversation visible for registered users only'))
-				} else if (listable.value === CONVERSATION.LISTABLE.ALL) {
-					showSuccess(t('spreed', 'You made the conversation visible for everyone'))
+				if (this.lastNotification) {
+					this.lastNotification.hideToast()
+					this.lastNotification = null
+				}
+				if (listable === CONVERSATION.LISTABLE.NONE) {
+					this.lastNotification = showSuccess(t('spreed', 'You made the conversation accessible to participants'))
+				} else if (listable === CONVERSATION.LISTABLE.USERS) {
+					this.lastNotification = showSuccess(t('spreed', 'You made the conversation accessible to registered users only'))
+				} else if (listable === CONVERSATION.LISTABLE.ALL) {
+					this.lastNotification = showSuccess(t('spreed', 'You made the conversation accessible to everyone'))
 				}
```

Comment:
there is a system message in the chat. shouldn't really spam with a success notification on top right helping people to get blind to notifications

Reply:
Only applies if you actually see the system message.
In the case of new conversation creation, there should be no such system message shown currently.

When using the settings dialog, if you change the dropdown it will save directly without closing the dialog, but you don't necessarily see the chat messages from there.

In general I think it's a good idea to have some kind of visual feedback for autosave fields. Here this is achieved with a notification. If we can come up with a different approach like have tick marks or so (system wide, as design guideline), it might work better. @jancborchardt thoughts ?

Reply:
The visual feedback should be the chat, or the checkmark goes to spinner and only turns into a checkmark/empty once the response is there. Otherwise you see 5 popup notifications when you organize a meeting and it's just making you blind.

Reply:
checkmark going to spinner is an idea, with a minimum delay in case it's too fast, so it looks like it's saving (I recently read about UI improvements where delays were added to give the user the impression that it's properly computing/saving)